### DISCUSSION TITLE
Which one of the following elements occurs most abundantly in our universe?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#682] [Galaxy Trivia] [answered]
* Nitrogen
* Hydrogen
* Helium

Ans only with the symbol and atomic no.

**Comment by ParanoidUser** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5120290/236318621-d6a5b466-dc35-40a8-ab80-77de5983d4a1.png)
